site/src/components/navigation/navigation.astro

```diff
@@ -58,7 +58,7 @@ const routesToNotPrefetch = ['/research/component-matrix']
   }
 </style>
 
-<NavigationContainer client:load>
+<NavigationContainer client:only>
   <CommunityLinks className="mobile" />
```

Comment:
Does this mean the JS for `CommunityLinks` runs client-side too? It inlines a bunch of SVGs into JSX, definitely want to avoid having it in any client bundles.

Reply:
Good point will need to double check that.

Reply:
So there's 4 inlined SVGs into the JSX but that's the case anyway. Seems fine?

Reply:
Probably better off converting them into a sprite sheet & consuming with `<use>`.

Not that a few SVGs is the end of the world but that's objectively the most expensive way to use them & by a large margin too.